packages/origin-ui-core/webpack/base.config.js

```diff
@@ -1,8 +1,11 @@
 const ExtractTextPlugin = require('extract-text-webpack-plugin');
 const HtmlWebpackPlugin = require('html-webpack-plugin');
 const CopyWebpackPlugin = require('copy-webpack-plugin');
+const Dotenv = require('dotenv-webpack');
```

Comment:
This whole env thing is not necessary, we approached env settings in a different way. We are using this script https://github.com/energywebfoundation/origin/blob/f6d77f327a7676c3e742cc8a022e5c085cf66e39/packages/origin-ui/env.sh to generate `env-config.js` file which then is read by the UI.

The main reason was to have `.env` variable available on the runtime, including edits. Otherwise full UI repack is required to have your new `.env` variables available.